packages/express/src/middleware/index.ts

```diff
@@ -33,14 +34,21 @@ export function expressMiddleware(appsignal: Appsignal): RequestHandler {
       res.end = function (this: Response) {
         res.end = originalEnd
 
-        const { method = "GET" } = req
+        const { method = "GET", params = {}, url } = req
+        const { query } = parse(url)
 
         // if there is no error passed to `next()`, the span name will
         // be updated to match the current path
         if (req.route?.path) {
           span.setName(`${method} ${req.route.path}`)
         }
 
+        // set route params (if parsed by express correctly)
+        span.setSampleData("params", {
+          ...params,
+          query: query || ""
```

Comment:
Would this include the whole, unfiltered query string? If so, this could include sensitive or identifiable information we can't easily filter out. Do we get the GET params in the main `params` variable as well? Since that's filtered by the agent, that might be the safer place for it.

Reply:
We should indeed use the filtered data variant here. Do we?

Reply:
Thanks @jeffkreeftmeijer @jeffkreeftmeijer, forgot about the data sanitization here. I have now changed this to combine `req.params` and `req.query` together as params, which needed a small refactor. Please re-review.

Reply:
Awesome, looking good!